Modules/_csv.c

```diff
@@ -12,6 +12,7 @@ module instead.
 
 #include "Python.h"
 #include "structmember.h"
+#include <stdbool.h>
```

Comment:
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/stdbool.h.html

How about using `bool` instead of `char`?  Isn't it portable?

Reply:
T_BOOL requires `char`. I have doubts that `_Bool` has always the same size and alignment as `char`.

Reply:
make sense.